### PR TITLE
Update remaining notebooks for release 0.1b2

### PR DESCRIPTION
**Pull request by BryanCutler**

### CODE REVIEW

File: notebooks/Analyze_Text.ipynb [hunk 1]
```diff
@@ -143,7 +143,7 @@
     {
```
Comment (BryanCutler) [inline notebooks/Analyze_Text.ipynb:143]: <p>Seems like an improvement from previous results, probably from updated Watson?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

File: notebooks/Integrate_NLP_Libraries.ipynb [hunk 2]
```diff
@@ -174,7 +174,7 @@
     {
```
Comment (BryanCutler) [inline notebooks/Integrate_NLP_Libraries.ipynb:174]: <p>This adds our new mentions DataFrame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (BryanCutler) [inline notebooks/Integrate_NLP_Libraries.ipynb:174]: <p>'along' changes fro <code>ADV</code>  -&gt; <code>ADP</code>  , my grammar isn't great - is that wrong now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

File: notebooks/Integrate_NLP_Libraries.ipynb [hunk 3]
```diff
@@ -2594,7 +2594,7 @@
        "    <tr>\n",
        "      <th>69</th>\n",
        "      <td>[288, 297): 'Appearing'</td>\n",
-       "      <td>NOUN</td>\n",
+       "      <td>PROPN</td>\n",
```
Comment (BryanCutler) [inline notebooks/Integrate_NLP_Libraries.ipynb:2597]: Seems wrong

File: notebooks/Integrate_NLP_Libraries.ipynb [hunk 4]
```diff
@@ -2699,15 +2699,15 @@
        "66        [280, 283): 'Sir'          PROPN    sir   \n",
        "67        [284, 287): 'Not'            ADV    not   \n",
        "68          [287, 288): '-'          PUNCT   None   \n",
-       "69  [288, 297): 'Appearing'           NOUN   None   \n",
+       "69  [288, 297): 'Appearing'          PROPN   None   \n",
```
Comment (BryanCutler) [inline notebooks/Integrate_NLP_Libraries.ipynb:2702]: seems wrong

**Comment by BryanCutler** [comment]
@frreiss these are the results of running the remaining notebooks from the current master, in preparation for the release. Most of the changes see fine to me, but there are a couple Watson results that seem sketchy.

File: notebooks/Analyze_Text.ipynb (hunk 1 above)
Comment (frreiss) [reply notebooks/Analyze_Text.ipynb:143]: <p>I think it's the same answer but in a different order. NLU isn't supposed to change results if you don't change the value of the <code>version</code>  argument (see the first code cell).</p>

Comment (frreiss) [reply notebooks/Analyze_Text.ipynb:143]: <p>On second, thought, looks like the confidence values have changed. Not sure what's going on. Maybe <code>&nbsp;version="2019-07-12"</code>  gives you a different actual version of the models now...</p>

File: notebooks/Integrate_NLP_Libraries.ipynb (hunk 2 above)
Comment (frreiss) [reply notebooks/Integrate_NLP_Libraries.ipynb:174]: <p>I believe that <code>ADP</code>  is the right answer here. The tagging system in NLU is based on Universal Dependencies tags iirc; see http://universaldependencies.org/docs/en/pos/all.html</p>

File: notebooks/Integrate_NLP_Libraries.ipynb (hunk 4 above)
Comment (frreiss) [reply notebooks/Integrate_NLP_Libraries.ipynb:2702]: All the models I've seen have had trouble with "Sir Not-Appearing-In-This-Film". I'll chalk that up to a lack of subtle British humor in the training data ;)

**Comment by BryanCutler** [comment]
Thanks @frreiss , I'll get started on making the release tomorrow

**Pull request merged**